src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', () => {
+    resolve('First promise was resolved');
```

Comment:
it should be resolved only on the **left** button click:
![image](https://user-images.githubusercontent.com/86923089/209664561-2cdc2042-f1c7-4e66-a3e9-328db66b4785.png)

Reply:
Yes, I saw it. But tests sad me that not right ((

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/79605408/209665881-1e9b4348-5358-498f-9552-9d8a7d2ddc49.png">


> it should be resolved only on the **left** button click: ![image](https://user-images.githubusercontent.com/86923089/209664561-2cdc2042-f1c7-4e66-a3e9-328db66b4785.png)

Reply:
I see
Then it's mismatching between tests and description - not your fault
So I'll approve the PR